index.html

```diff
@@ -2150,6 +2150,61 @@ <h3 id="operators-partial">Partial Application, Operators as Functions</h3>
         </div>
       </div>
 
+      <a name="export"></a>
+      <h3 id="operators-export">export</h3>
+      <p>By using the <code>export</code> operator instead of <code>exports</code> you get a more concise way to define modules.</p>
+
+      <div class="example">
+        <div class="example-ls">
+<pre class="prettyprint lang-ls">
+
+
+
+export 123
+
+export value
+
+export num:123
+
+export func:->
+
+export 
+  a:1
```

Comment:
The example was 

    export 
      a:1
      b:->

should I change it to 

    export a:1 b:->

?